rubicon/objc/collections.py

```diff
@@ -168,25 +168,142 @@ def _find(self, sub, start=None, end=None, *, reverse):
         else:
             return found_range.location
 
-    def find(self, sub, start=None, end=None):
-        return self._find(sub, start=start, end=end, reverse=False)
-
-    def rfind(self, sub, start=None, end=None):
-        return self._find(sub, start=start, end=end, reverse=True)
-
     def _index(self, sub, start=None, end=None, *, reverse):
         found = self._find(sub, start, end, reverse=reverse)
         if found == -1:
             raise ValueError('substring not found')
         else:
             return found
 
+    def capitalize(self):
+        return str(self).capitalize()
+
+    def casefold(self):
+        return str(self).casefold()
+
+    def center(self, width, fillchar=' '):
```

Comment:
I thought about taking the `*args, **kwargs` approach, but decided against it because it would obscure the underlying prototype from ipython etc. 

The `__getattr__` call is an interesting idea, though - it's a lot less effort to implement, which might make the loss of detail worth it...